steppy/base.py

```diff
@@ -539,25 +622,28 @@ def _format_step_name(self, name, transformer):
             name = str(name)
         else:
             name = transformer.__class__.__name__
-        return '{}{}'.format(name, self._get_step_suffix(name))
+        return name
 
-    def _validate_step_name(self, name):
+    @staticmethod
+    def _validate_step_name(name):
```

Comment:
private staticmethod seems weird I would choose either one or the other.

Reply:
agree, use private method